src/fluree/server/consensus/events.clj

```diff
@@ -225,4 +229,12 @@
   [evt]
   (or (transaction-committed? evt)
       (ledger-created? evt)
+      (ledger-dropped? evt)
       (error? evt)))
+
+(defn watcher-id
```

Comment:
This is minor, but I think we should update our current code to use this new function instead of using the tx-id directly.